### ISSUE TITLE
File history does not display authors properly 

### ISSUE THREAD

**Issue by AntoineDupre** [Scille/parsec-cloud#1270]
File history does not display authors properly 
![history](https://user-images.githubusercontent.com/1070501/87525568-f74c1880-c689-11ea-85cc-96d9dd0ebfd8.png)

**Comment by Max-7** [comment]
VersionLister.list() returns a UserID instead of a UserInfo.

We could use LoggedCore.get_user_info() to retrieve the human handle from the UserID but it seems to me that it would be way better to have VersionLister.list() return the UserInfo directly as it would seriously limit the number of queries made to the backend.

**Comment by bitlogik** [comment]
We made some tests on Parsec 12 days ago, and we noticed that. As there are many work in progress on this topic we were not sure if this was a bug or just a remaining task to clear. Author and device are not "decoded" in the file history listing.

When you wrote "we could use LoggedCore.get_user_info() from the backend", I thought that the main goal of this obfuscated id, was to hide the real info from the server, so that the server never knows the info names. Did I miss anything?
Indeed, on top of  avoiding a query to the server to get a content which is known locally, for the sake of logic, this user info should be get locally, not from the server.
